cmd/fleetctl/api.go

```diff
@@ -58,3 +65,26 @@ func clientFromCLI(c *cli.Context) (*service.Client, error) {
 
 	return fleet, nil
 }
+
+func setupRootCA(client *service.Client, rootca string) *service.Client {
+	// get system pool, use empty on error
+	rootCAs, _ := x509.SystemCertPool()
```

Comment:
Can we reason a bit more about whether this functionality should _append_ a cert to the system pool, or use _only_ the certs provided in the pem? I think it may be better to use only the provided cert (and error if this is missing) because it effectively allows pinning of the root.